src/styles/components/table.scss

```diff
@@ -0,0 +1,64 @@
+@import "../core/settings";
+@import "../core/mixins";
+@import "../core/functions";
+
+$block: ns(table);
+
+/*!
+    .tn-table
+        .tn-table__row, tn-table__row--stripe | .is-active
+        .tn-table__cell
+*/
+
+// TABLE
+//     - not-striped
+
+
+
+.#{$block} {
+    //BASE
+    //set all BLOCK reset and baseline styles
+    @include reset;
+    border-collapse: collapse;
+    width: 100%;
+    background-color: color(light);
+
+    //VARIANTS
+    //each variant MUST extend base block
+    // &--modifier {
+    //     @extend .#{$block};
+    // }
```

Comment:
This is here as an example to show how a dev would need to extend the block when adding a modifier. I can remove it and create an example SCSS file for reference.

NOTE: This is only necessary to avoid duplicating classes like `<table class="tn-table tn-table--foo">`. If we want to build the library to support  `<table class="tn-table--foo">` where the modified class get all the base rules , then we have to code it like the example. My alternative approach was squashed in #663.